lib/logstash/environment.rb

```diff
@@ -77,7 +77,7 @@ def jruby?
     end
 
     def windows?
-      Gem.win_platform?
+      ::Gem.win_platform?
```

Comment:
this is not really related to this PR but do we know if there is a particular reason to use `Gem` here instead of relying on the `RUBY_PLATFORM` constant?